roles/usegalaxy-eu.bashrc/templates/get_jwd.sh.j2

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Description: Get the job working directory using the job id and the object store configuration XML file
```

Comment:
You have a python script for that, isn't it? How does this script relate to it?

Reply:
I have a Python script for cleaning up JWD's. I can also tweak that script and this functionality to it, and we can call a sub-command `get_jwd` via the command line that would return the JWD path and another sub-command `cleanup_jwd` that would clean up the JWDs if you would like.

Reply:
I think it would make sense to have this logic only in one place and not in two different scripts. Especially if you are planning to upstream your python script, we could reuse it easily.

Reply:
I have totally updated the script. Here is the [gist](https://gist.github.com/sanjaysrikakulam/5c90ef4ddd380d3196799a4215b0f1f2). Please have a look and share your review.

Reply:
@sanjaysrikakulam this looks largely ok, just a few small things. I guess its ready for trying to upstream it.

Reply:
@bgruening Created PR: https://github.com/galaxyproject/galaxy/pull/15618

Reply:
Shall I remove this bash script and replace it with the python script (in the above PR) until the upstream merge it and release it?

Reply:
Sounds good!

Reply:
I have updated this PR. Please review everything and let me know if any changes are required.